pkg/manager/manifests/templates/deployment.yaml

```diff
@@ -32,6 +32,11 @@ spec:
           - "--addon-namespace={{ .AddonInstallNamespace }}"
           - "--with-image-override={{ .HyeprshiftImageOverride }}"
           - "--hypershfit-operator-image={{ .HypershiftOperatorImage }}"
+          - "--multicluster-pull-secret= {{ .MulticlusterEnginePullSecret }}"
```

Comment:
@zhujian7 could we remove the extra blank here? like change the 
`"--multicluster-pull-secret= {{ .MulticlusterEnginePullSecret }}"`
to 
`"--multicluster-pull-secret={{ .MulticlusterEnginePullSecret }}"`

Reply:
Thanks for the review, https://github.com/stolostron/hypershift-addon-operator/pull/41